### DISCUSSION TITLE
build this project use goreleaser in my meachine,it's size so big, reach 20MB +

### DISCUSSION THREAD

**Discussion by Fly-Playgroud** [projectdiscovery/katana#315] [Q&A] [answered]
build this project use goreleaser in my meachine,it's size so big, reach 20MB +, but download binary from from github release,it's so tiny, just 7MB

**Comment by ehsandeep** [comment, marked as answer]
@Fly-Playgroud we are using `goreleaser/goreleaser-action@v4` GitHub action to build binary on GitHub.
